packages/core/src/Site/index.js

```diff
@@ -1313,16 +1313,17 @@ class Site {
       if (!remoteUrl) {
         return null;
       }
+      const parts = remoteUrl.split('/');
       if (remoteUrl.startsWith(HTTPS_PREAMBLE)) {
         // https://github.com/<name|org>/<repo>.git (HTTPS)
-        const parts = remoteUrl.split('/');
-        const repoName = parts[parts.length - 1].toLowerCase();
+        const repoNameWithExt = parts[parts.length - 1];
+        const repoName = repoNameWithExt.substring(0, repoNameWithExt.lastIndexOf('.'));
```

Comment:
hmm what is the new `.lastIndexOf('.')` for? (and removal of `toLowerCase`)

Reply:
since repoName would be something like `myrepo.git` (since `parts` is something like ['https', ..., myrepo.git']), in order to generate a correct website link the `.git` should be stripped, which I didn't do before.

As for the removal of toLowerCase, I discovered that the repository name would be directly used in the gh-pages url, so if I have a repository `Orbital3` my URL would be `https://kendrickang.github.io/Orbital3/` --> then .toLowerCase() produce `https://kendrickang.github.io/orbital3/`, which would be incorrect.

I believe this wasn't found in my testing when I wrote this because I only explicitly tested for when CNAME was present (whereas this block is only run when CNAME isn't present)

Reply:
thanks for the explanation! I'm not opposed to having these fixes here as well since its deploy url related too.
Let's change the commit message title and body accordingly to reflect the changes here though, in case anyone wonders what the changes are for

Reply:
done!